src/components/NavBar/SearchBar.tsx

```diff
@@ -156,9 +156,10 @@ export const SearchBar = () => {
       >
         <Row
           className={clsx(
-            ` ${styles.nftSearchBar} ${!isOpen && !isMobile && magicalGradientOnHover} ${
-              isMobileOrTablet && (isOpen ? styles.visible : styles.hidden)
-            } `
+            ` ${styles.nftSearchBar} 
+              ${!isOpen && !isMobile && magicalGradientOnHover} 
+              ${isMobileOrTablet && (isOpen ? styles.visible : styles.hidden)} 
+              `
```

Comment:
eslint was yelling at me ...

Reply:
if you use clsx you can just pass in arguments for each of these. don't need this string interpolation.